crates/services/sync/src/ports.rs

```diff
@@ -31,6 +31,14 @@ pub trait PeerToPeerPort {
         height: BlockHeight,
     ) -> anyhow::Result<Option<SourcePeer<SealedBlockHeader>>>;
 
+    /// Request range of sealed block headers from the network
+    /// starting from block height `start` up to, and including `end`.
+    async fn get_sealed_block_headers_inclusive(
+        &self,
+        start: BlockHeight,
+        end: BlockHeight,
```

Comment:
Maybe it is better to work with the `Range` directly

Reply:
Range of `u32`?

Reply:
I prefer using domain types. I think `Range<BlockHeight>` might be a thing, but it doesn't implement `Iterator` for example, so it's not great to work with.

Reply:
You can work with `Range<u32>` for now. Leave todo to replace it with `BlockHeight` and implement `Iterator` in the `fuel-type` crate in parallel or follow-up=)